docs/source/markdown/podman-create.1.md

```diff
@@ -409,6 +409,16 @@ millions of trillions.
 
 Add metadata to a container (e.g., --label com.example.key=value)
 
+Users can set a special label **io.containers.capabilities=CAP1,CAP2,CAP3** that
```

Comment:
I feel like this belongs elsewhere. We don't expect people to set this at `podman run` time, right? Instead we want them to load it from the image.

Should we instead be setting `io.containers.capabilities` at container creation time based on the capabilities we put in the spec, so that it will be there when we commit images?

Reply:
No the goal here is to override the defaults with something more secure.

Reply:
But rather than trying to get people to add the label themselves, we can make it for them from what they entered in `--cap-add` and `--cap-drop`. We know the final capabilities the container was created with, so it's pretty trivial to make the label from that.

Reply:
But that is not natural behaviour for Building container images.  Most people, if they are going to use this feature are going to do it via a Dockerfile.  We could add the label as a side effect of --cap-add and --cap-drop, but I am not sure that would be expected by users.

Reply:
I agree that most people will be using a Dockerfile - I'm mostly confused as to why people would ever want to set it directly at `podman run` time, instead of using `--cap-add` and `--cap-drop`.

Reply:
Sure, but do you just want me to not document that you can do this?